### PR TITLE
Restructure features pages

### PR DESCRIPTION
**Pull request by karlchillmaid**
This pull request updates our features pages to complement the new ‘Using Notify’ guidance we’ve published.

Explanatory content has moved to the ‘Using Notify’ pages, so we’re paring the Features pages back to a single page with a list of highlights. Users can click on the links for subjects of interest to discover more.

Changes:
* update the content for the main features page, including links to Using Notify and Pricing pages
* delete the features sub-pages for Email, Text messages and Letters
* update routes and navigation subdictionaries

Some link text or destination page headings may need to change, but that will come in a later PR when I go through and resolve any content debt.

### CODE REVIEW

File: app/templates/views/features.html [hunk 1]
```diff
@@ -10,69 +10,64 @@
...
+<p class="govuk-body">Notify commits to:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>sending 95% of emails and text messages within 10 seconds</li>
+  <li>printing and posting letters by 3pm the next working day</li>
+</ul>
+<p class="govuk-body">Find out more about <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.performance') }}">GOV.UK Notify’s performance</a>.</p>
 
 <h2 class="heading-medium" id="support">Support</h2>
-<p class="govuk-body">Notify provides 24-hour online support. If you have an emergency outside office hours, we’ll reply within 30 minutes.</p>
+<p class="govuk-body">Notify provides 24-hour online support for emergencies. If you have an emergency outside office hours, we’ll reply within 30 minutes.</p>
```
Comment (karlchillmaid) [inline app/templates/views/features.html:70]:
```suggestion
<p class="govuk-body">GOV.UK Notify provides 24-hour online support for emergencies. If you have an emergency outside office hours, we’ll reply within 30 minutes.</p>
```

File: app/templates/views/features.html [hunk 2]
```diff
@@ -10,69 +10,64 @@
...
-<p class="govuk-body">Read more about <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.security') }}">security</a>.
+<h2 class="heading-medium">Performance</h2>
+<p class="govuk-body">Notify commits to:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>sending 95% of emails and text messages within 10 seconds</li>
+  <li>printing and posting letters by 3pm the next working day</li>
+</ul>
+<p class="govuk-body">Find out more about <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.performance') }}">GOV.UK Notify’s performance</a>.</p>
 
 <h2 class="heading-medium" id="support">Support</h2>
```
Comment (karlchillmaid) [inline app/templates/views/features.html:69]:
```suggestion
<h2 class="heading-medium" id="support">24-hour online support</h2>
```

File: app/templates/views/features.html [hunk 3]
```diff
@@ -10,69 +10,64 @@
...
+<ul class="govuk-list govuk-list--bullet">
+  <li>sending 95% of emails and text messages within 10 seconds</li>
+  <li>printing and posting letters by 3pm the next working day</li>
+</ul>
+<p class="govuk-body">Find out more about <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.performance') }}">GOV.UK Notify’s performance</a>.</p>
 
-<h2 class="heading-medium" id="support">Support</h2>
-<p class="govuk-body">Notify provides 24-hour online support. If you have an emergency outside office hours, we’ll reply within 30 minutes.</p>
+<h2 class="heading-medium" id="support">24-hour online support</h2>
+<p class="govuk-body">GOV.UK Notify provides 24-hour online support for emergencies. If you have an emergency outside office hours, we’ll reply within 30 minutes.</p>
```
Comment (karlchillmaid) [inline app/templates/views/features.html:70]:
```suggestion
<p class="govuk-body">If you have an emergency outside office hours, we’ll reply within 30 minutes.</p>
```

File: app/templates/views/features.html [hunk 4]
```diff
@@ -10,69 +10,64 @@
...
-<h3 class="heading-small">Hide sensitive information</h3>
-<p class="govuk-body">Notify lets you redact personal information from your messages after they have been sent. This means that only the recipient can see that information.</p>
+<h2 class="heading-medium">Security</h2>
+<p class="govuk-body">Find out how <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.security') }}">GOV.UK Notify protects and manages your data</a>.</p>
 
-<h3 class="heading-small">Two-factor authentication</h3>
-<p class="govuk-body">Notify uses two-factor authentication (2FA) to keep your account secure. When you sign in, we’ll send a unique one-time code to your phone and ask you to enter it before we let you use your account.</p>
-<p class="govuk-body">Read more about <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.security') }}">security</a>.
+<h2 class="heading-medium">Performance</h2>
+<p class="govuk-body">Notify commits to:</p>
```
Comment (karlchillmaid) [inline app/templates/views/features.html:62]:
```suggestion
<p class="govuk-body">GOV.UK Notify commits to:</p>
```

File: app/templates/views/features.html [hunk 5]
```diff
@@ -10,69 +10,65 @@
...
 
-<h2 class="heading-medium" id="performance">Performance</h2>
-<p class="govuk-body">Notify commits to:</p>
+<h2 class="heading-medium">Send and receive text messages</h2>
 <ul class="govuk-list govuk-list--bullet">
-  <li>sending 95% of emails and text messages within 10 seconds</li>
-  <li>printing and posting letters by 3pm the next working day (if you send them to us before 5:30pm)</li>
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.pricing', _anchor='text-messages') }}">Send thousands of free text messages</a></li>
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_receive_text_messages') }}">Receive text messages</a></li>
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_text_message_sender') }}">Tell recipients who your messages are from</a></li>
```
Comment (karlchillmaid) [inline app/templates/views/features.html:44]:
```suggestion
  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_text_message_sender') }}">Tell recipients who your messages are from</a></li>
    <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.pricing'), _anchor='international-numbers') }}">Send text messages to international numbers</a></li>
```

File: app/templates/views/features.html [hunk 6]
```diff
@@ -10,69 +10,66 @@
...
+    <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.pricing'), _anchor='international-numbers') }}">Send text messages to international numbers</a></li>
```
Comment (karlchillmaid) [inline app/templates/views/features.html:45]:
```suggestion
    <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.pricing'), _anchor='international-numbers') }}">Send text messages to international numbers</a></li>
  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_text_message_sender') }}">Tell recipients who your messages are from</a></li>
```

File: app/templates/views/features.html [hunk 7]
```diff
@@ -10,69 +10,66 @@
...
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.pricing', _anchor='text-messages') }}">Send thousands of free text messages</a></li>
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_receive_text_messages') }}">Receive text messages</a></li>
+    <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.pricing'), _anchor='international-numbers') }}">Send text messages to international numbers</a></li>
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_text_message_sender') }}">Tell recipients who your messages are from</a></li>
 </ul>
 
-<p class="govuk-body">We send messages through several different providers. If one provider fails, Notify switches to another so that your messages are not affected.</p>
-
-<p class="govuk-body">Visit our <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.performance') }}">performance data</a> page to see how Notify is performing.</p>
+<h2 class="heading-medium">Print and post letters</h2>
```
Comment (karlchillmaid) [inline app/templates/views/features.html:48]:
```suggestion
<h2 class="heading-medium">Let Notify print and post your letters</h2>
```

File: app/templates/views/features.html [hunk 8]
```diff
@@ -10,69 +10,66 @@
...
+<ul class="govuk-list govuk-list--bullet">
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.pricing'), _anchor='letters') }}">Choose your postage</a></li>
+  <li>Send international letters</li>
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_letter_branding') }}">Add letter branding</a></li>
+  <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.guidance_upload_a_letter') }}">Upload a letter from a PDF</a></li>
+</ul>
 
-<h2 class="heading-medium" id="security">Security</h2>
-<p class="govuk-body">Notify protects and manages data to meet the needs of government services.</p>
+<h2 class="heading-medium">Automated sending</h2>
```
Comment (karlchillmaid) [inline app/templates/views/features.html:56]: I wonder if ‘automatic’ reads more simply than ‘automated’

```suggestion
<h2 class="heading-medium">Automatic sending</h2>
```

Comment (saimaghafoor) [reply app/templates/views/features.html:56]: I like automatic - to me it emphasises a user not having to actually do anything more than automated does and is probably more recognisable than automated.

File: app/templates/views/features.html (hunk 7 above)
Comment (saimaghafoor) [reply app/templates/views/features.html:48]: 'Notify prints and posts your letters'

**Comment by saimaghafoor** [comment]
Looks good. Added a couple of comments for content above,

**Review by saimaghafoor — APPROVED** [review]

**Comment by karlchillmaid** [comment]
I think we need to redirect the 3 features sub-pages to the main features page. Could do with a hand adding that to app/main/views/index.py

File: app/templates/views/features.html [hunk 9]
```diff
@@ -10,69 +10,66 @@
...
-<h2 class="heading-medium" id="security">Security</h2>
-<p class="govuk-body">Notify protects and manages data to meet the needs of government services.</p>
+<h2 class="heading-medium">Automatic sending</h2>
+<p class="govuk-body">To send messages automatically, you’ll need a developer to integrate with the GOV.UK Notify API.</p>
+<p class="govuk-body">For more information, read our <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.documentation') }}">API documentation</a>.</p>
 
-<h3 class="heading-small">Hide sensitive information</h3>
-<p class="govuk-body">Notify lets you redact personal information from your messages after they have been sent. This means that only the recipient can see that information.</p>
+<h2 class="heading-medium">Keep your data secure</h2>
+<p class="govuk-body">Read about <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.security') }}">security on GOV.UK Notify</a>.</p>
```
Comment (karlchillmaid) [inline app/templates/views/features.html:61]:
```suggestion
<p class="govuk-body">Find out more about <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.security') }}">security on GOV.UK Notify</a>.</p>
```

File: app/templates/views/features.html [hunk 10]
```diff
@@ -10,69 +10,66 @@
...
+  <li>sending 95% of emails and text messages within 10 seconds</li>
+  <li>printing and posting letters by 3pm the next working day</li>
+</ul>
+<p class="govuk-body">See the latest <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.performance') }}">performance data</a>.</p>
 
-<h2 class="heading-medium" id="support">Support</h2>
-<p class="govuk-body">Notify provides 24-hour online support. If you have an emergency outside office hours, we’ll reply within 30 minutes.</p>
+<h2 class="heading-medium" id="support">24-hour online support</h2>
+<p class="govuk-body">If you have an emergency outside office hours, we’ll reply within 30 minutes.</p>
 <p class="govuk-body">Find out more about <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.support') }}">support</a>.</p>
```
Comment (karlchillmaid) [inline app/templates/views/features.html:73]:
```suggestion
<p class="govuk-body"><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.support') }}">Contact the GOV.UK Notify team</a>.</p>
```

**Review by leohemsted — APPROVED** [review]
Happy with this from a code/dead links/etc perspective

**Pull request merged**